docs/installing.rst

```diff
@@ -148,15 +148,36 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
------------------------------------------------------
-3. Install the vagrant-service-manager Vagrant plugin
------------------------------------------------------
+----------------------------------
+3. Install additional dependencies
+----------------------------------
+
+For some operating systems, you might need to install additional dependencies before you install the Vagrang plugins.
+
+* Fedora 24
+
+  Run the following commands to install the additional dependencies::
+
+  $ sudo dnf install "Development Tools"
+  $ sudo dnf install rpm-build
+  $ sudo dnf install zlib-devel
+  $ sudo dnf install ruby-devel
+
+--------------------------
+4. Install Vagrant plugins
+--------------------------
 
-The `vagrant-service-manager`_ plugin can be installed using::
+Run the following commands to install the required Vagrant plugins::
 
-    vagrant plugin install vagrant-service-manager
+$ vagrant plugin install vagrant-registration
```

Comment:
AFAIK, vagrant-registration is not required for the ADB.  Since CDK docs are currently separate, I believe it should be deleted.

@LalatenduMohanty @hferentschik